Formula/xcinfo.rb

```diff
@@ -0,0 +1,20 @@
+class Xcinfo < Formula
+  desc "Tool to get information about and install available Xcode versions"
+  homepage "https://github.com/xcodereleases/xcinfo"
+  url "https://github.com/xcodereleases/xcinfo/archive/0.5.0.tar.gz"
+  sha256 "07b6c2f5a5b6a5f3cd6d190ac16c36c5011a87f2a1da2a109ccaa909eb6bcc2c"
+  license "MIT"
+
+  depends_on xcode: "12.0"
```

Comment:
Does it need this at runtime?

Reply:
Well, actually it need macOS 10.15 (Catalina) or 11 (Big Sur) to run and Swift 5.2 to compile.

Reply:
Is there a possibility to describe this?

Reply:
The second one would have to be an `xcode` dependency like this. And the first is `depends_on macos: :catalina`